hack/tests/e2e-helm.sh

```diff
@@ -49,7 +49,7 @@ test_operator() {
     then
         kubectl describe pods -l "app.kubernetes.io/instance=${release_name}"
         kubectl describe deployments ${nginx_deployment}
```

Comment:
This is still okay? 

What was the reason behind change for the below, I am not aware of the problem in the first place.

Reply:
The issue was that the we weren't actually running a loop to test the condition in lines 62 and 73. They just tested the condition once and would fail out immediately.

The log lines were changed because they were supposed to be for the nginx-operator deployment all along. The logs from the nginx pods spun up by the operator aren't useful for debugging test failures.